src/scss/styles.scss

```diff
@@ -0,0 +1,135 @@
+@import "../../node_modules/bootstrap/scss/bootstrap";
+@import "variables";
```

Comment:
To customize bootstrap using SASS, the import of your custom variable should come before the import of Bootstrap